code/libraries/koowa/libraries/database/rowset/abstract.php

```diff
@@ -558,9 +558,19 @@ public function setTable($table)
     public function toArray()
     {
         $result = array();
+
         foreach ($this as $key => $row) {
             $result[$key] = $row->toArray();
+
```

Comment:
Why do you need this? If the toArray method of ModelEntity or DatabaseRowset include collecting the computed properties, why does the towery need to do it also?